packages/api/cms-api/src/dam/blocks/dam-file-download-link-block-transformer.service.ts

```diff
@@ -32,12 +33,14 @@ export class DamFileDownloadLinkBlockTransformerService implements BlockTransfor
                 id: file.id,
                 name: file.name,
                 fileUrl: await this.filesService.createFileUrl(file, { previewDamUrls, relativeDamUrls }),
+                size: Number(file.size),
```

Comment:
@thomasdax98 for some reason (probably due to [BigIntType](https://mikro-orm.io/api/core/class/BigIntType)) this is a string when loading the file. I added the Number cast as a quick workaround. We should fix this in a follow-up PR